### ISSUE TITLE
Error message when using create

### ISSUE THREAD

**Issue by yzcj105** [Glavo/gchecksum#2]
time gchecksum c  -f ../checksums.md5     
 
```                                          
java.util.concurrent.ExecutionException: java.io.IOException: Is a directory
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
	at org.glavo.checksum.mode.CreateOrUpdate.lambda$update$3(CreateOrUpdate.java:171)
	at java.util.TreeMap.forEach(TreeMap.java:1005)
	at org.glavo.checksum.mode.CreateOrUpdate.update(CreateOrUpdate.java:169)
	at org.glavo.checksum.Main.main(Main.java:302)
Caused by: java.io.IOException: Is a directory
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.read(FileDispatcherImpl.java:46)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:159)
	at org.glavo.checksum.Hasher.hashFile(Hasher.java:129)
	at org.glavo.checksum.mode.CreateOrUpdate$2.lambda$processFile$0(CreateOrUpdate.java:164)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Done
```

**Comment by Glavo** [comment]
Do you have symbolic link in the directory? I released gchecksum 0.11.0, which follows the symbolic link.

**Comment by yzcj105** [comment]
Yes. It contains soft links.  Thanks for the quick reply.  I will try the new 0.11 version.
